Move.java

```diff
@@ -0,0 +1,3 @@
+enum Move {
+	LEFT, RIGHT, UP, DOWN, LEFT_UP, LEFT_DOWN, RIGHT_UP, RIGHT_DOWN
```

Comment:
You could also include the x and y offsets in each of these enum values like
LEFT(0, -1),
RIGHT(0, 1),

Then define the constructor Move(int yOffset, int xOffset)